mis_builder/models/accounting_none.py

```diff
@@ -167,9 +168,6 @@ def __repr__(self):
     def __str__(self):
         return ''
 
-    def __nonzero__(self):
-        return False
-
```

Comment:
If I read the [doc](https://docs.python.org/2/reference/datamodel.html#object.__nonzero__) correctly we need this for python 2. What is weird is that the test of this PR seems to pass, and we have a test for that...